app/models/instance.rb

```diff
@@ -82,6 +82,12 @@ def default?
     host == Instance::DEFAULT_HOST_NAME
   end
 
+  # Replace the hostname of the default instance.
+  def host
```

Comment:
@lowjoel I actually want to use a new method instead of overriding this, but cannot find a better name :( what do you suggest ?